.github/workflows/release.yml

```diff
@@ -162,23 +236,21 @@ jobs:
         with:
           go-version: ${{ env.GO_VERSION }}
 
-      - name: Create manifests
+      - name: Create KLT manifest
```

Comment:
cert manager manifest generation is removed from KLT manifest. not sure if this is the way to go or if KLT should have the full combined manifest

Reply:
IMHO KLT should have an entry in the manifests to deploy the cert-manager. 
What would you advise to handle the cert-manager version in KLT?
1. this entry in KLT should have a release-please tag so the cert-manager version is bumped as part of the cert-manager release PR
2. we use dependabot/rennovate to bump the cert-manager version

Reply:
i would go with renovate, since maybe we don't want to always immediately bump the cert manager version in KLT. with renovate we can control when we wanna do an upgrade and that way, it will also show up in the changelog

Reply:
I agree - with everything in the release-please PR, we could introduce a breaking change that would cause the pipeline to fail :/ beside the fact that it won't run since we reference an image that is not yet published 😅